packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -2048,6 +2048,30 @@ describe('cluster', () => {
     });
   });
 
+  test('create a cluster from a snapshot with single user secret rotation', () => {
```

Comment:
Thanks for adding this test. We need test cases for `addRotationMultiUser` as well, though.

Reply:
Will do